rankers/AggregateRanker/tests/test_aggregate_ranker.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
+
+from jina.executors.rankers import Chunk2DocRanker
+import pytest
+from .. import AggregateRanker
+
+
+def create_data():
```

Comment:
can we make it a `fixture`?

Reply:
thanks - it removes quite some lines - done